packages/shared/src/components/modals/SharePostModal.tsx

```diff
@@ -66,7 +66,7 @@ export default function SharePostModal({
     <ResponsiveModal
       padding={false}
       {...props}
-      style={{ content: { maxWidth: '26.25rem' } }}
+      contentClassName="!max-w-[26.25rem]"
```

Comment:
Do we need the ! important?

Reply:
Yes, we need, because responsive modal has max-width: 480px rule (and I guess this is why it was previously done will inline styles), but to convert to pixels and override the rule we need to use important